tests/sles4sap/netweaver_test_instance.pm

```diff
@@ -19,10 +19,6 @@ sub run {
 
     $self->select_serial_terminal;
 
-    # First, upload the installation logs
-    # NOTE: done here because there are multiple installation paths
```

Comment:
I wonder what was the reason to have this in `sles4sap/netweaver_test_instance` module, instead of in the installation module. It was introduced in #10308, but other than what is given as a reason in this comment, there is no extra discussion or explanation. Looking at the changes on the HANA modules in the same PR, I guess the multiple installation paths could refer to autoyast vs. wizard vs. regular installation, so my recommendation would be to add VR also for such tests.

It does make more sense to have this call in the installation module.

Reply:
> I wonder what was the reason to have this in `sles4sap/netweaver_test_instance` module, instead of in the installation module. It was introduced in #10308, but other than what is given as a reason in this comment, there is no extra discussion or explanation. Looking at the changes on the HANA modules in the same PR, I guess the multiple installation paths could refer to autoyast vs. wizard vs. regular installation, so my recommendation would be to add VR also for such tests.
> 
> It does make more sense to have this call in the installation module.

But we don't have tests installing Netweaver via AutoYaST or the Wizard.